lesson06/assignment/css/_components.content.scss

```diff
@@ -1,13 +1,25 @@
-html {
-    box-sizing: border-box;
+.navbar {
+    background-color: #e4cece;
 }
 
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
+#quoteCard {
+    width: 45rem;
 }
 
-.main {
-    padding: 2rem;
+#homeFeature {
+    width: 18rem;
```

Comment:
IDs on HTML elements should only be used for Javascript selection, not CSS selection. If you need an extremely specific selector, try using both the element and class, or a parent/child selector.